Formula/blast.rb

```diff
@@ -34,11 +44,26 @@ def install
   end
 
   test do
+    output = shell_output("#{bin}/update_blastdb.pl --showall")
+    assert_match "nt", output
+
+    # output = shell_output("#{bin}/get_species_taxids.sh -t 9606")
+    # assert_match "9606", output
+
     (testpath/"test.fasta").write <<~EOS
       >U00096.2:1-70
       AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC
     EOS
     output = shell_output("#{bin}/blastn -query test.fasta -subject test.fasta")
     assert_match "Identities = 70/70", output
+
+    # Create BLAST database
+    output = shell_output("#{bin}/makeblastdb -in test.fasta -out foo -title FOO -dbtype prot")
```

Comment:
```suggestion
    output = shell_output("#{bin}/makeblastdb -dbtype prot -in test.fasta -out test")
```
`-title` is optional and the default title is `test.fasta`.

Reply:
OK, thanks.